okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/InstrumentedInterceptor.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting3.okhttp;
+
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+import java.io.IOException;
+import okhttp3.Interceptor;
+import okhttp3.Response;
+
+/**
+ * Records metrics about the response codes of http requests.
+ */
+public final class InstrumentedInterceptor implements Interceptor {
```

Comment:
Curious to see what granularity you're tracking these -- per target service interface or per individual endpoint.

Reply:
currently interested in service level